ptp/c4u/clock/oscillatord.go

```diff
@@ -0,0 +1,61 @@
+/*
+Copyright (c) Facebook, Inc. and its affiliates.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package clock
+
+import (
+	"net"
+	"time"
+
+	osc "github.com/facebook/time/oscillatord"
+	ptp "github.com/facebook/time/ptp/protocol"
+)
+
+const (
+	oscillatordAddress = "127.0.0.1:2958"
```

Comment:
I was wondering if at this point it makes sense to put this default oscillatord monitoring port number into `time/oscillatord` const, as we already use it in few places.